samples/led/main.c

```diff
@@ -1,19 +1,12 @@
+#include <hal/led.h>
 #include <hal/xbox.h>
-#include <xboxkrnl/xboxkrnl.h>
-
-#define SMC_REG_LEDMODE             0x07
-#define SMC_REG_LEDSEQ              0x08
 
 void main(void)
 {
-    // Set LED to be user controlled
-    HalWriteSMBusValue(0x20, SMC_REG_LEDMODE, FALSE, 0x01);
+    // Set front LEDs to shine green for ~250 ms then red for ~250 ms roughly every second
```

Comment:
nit: Someone should confirm the timing; someone might use these exact values for handling serial communication

Reply:
A bit of a LED-timing-analysis (with cameras, oscilloscopes and logic analyzers) party started on Discord. - No consensus on timing was reached.

We should probably have https://xboxdevwiki.net/LED (or something) and link to that, to stop the bikeshed and prevent decentralizing information. Having a wiki article also leaves more room to explain how the LED behaves when opening the tray and so on.

We'll continue to bikeshed over this stuff for days otherwise.